internal/pkg/client/ocisif/ocisif.go

```diff
@@ -281,5 +283,18 @@ func PushOCISIF(_ context.Context, sourceFile, destRef string, ociAuth *ocitypes
 		return fmt.Errorf("while obtaining image: %w", err)
 	}
 
+	if ociAuthFile != "" {
```

Comment:
Instead of this block, we need to modify the AuthOptn function that is used below, so it can pull the credential out of a file at a custom path.

That code is in: https://github.com/preminger/singularity/blob/bfb7a87dd808dde5798da88cfe0061dcc209221c/internal/pkg/client/ocisif/auth.go#L102

Reply:
Underlying cause is #2226; marking as resolved.